scripts/ec2/allocate_address.coffee

```diff
@@ -0,0 +1,22 @@
+# Description:
+#   Create a new elastic IP
+#
+# Commands:
+#   hubot ec2 allocate address
+
+aws = require('../../aws').aws()
+ec2 = new aws.EC2()
```

Comment:
Creating an aws instance dynamically is necessary as other existing files to load latest environment values. This library depends on environment values, so we recommend to use [hubot-env](https://github.com/yoheimuta/hubot-aws#use-hubot-env) that load them dynamically.

```diff
# This should be applied for all 4 files.
diff --git a/scripts/ec2/allocate_address.coffee b/scripts/ec2/allocate_address.coffee
index 3226680..e07d5e9 100644
--- a/scripts/ec2/allocate_address.coffee
+++ b/scripts/ec2/allocate_address.coffee
@@ -4,11 +4,11 @@
 # Commands:
 #   hubot ec2 allocate address
 
-aws = require('../../aws').aws()
-ec2 = new aws.EC2()
 
 module.exports = (robot) ->
   robot.respond /ec2 allocate address$/i, (msg) ->
+    aws = require('../../aws').aws()
+    ec2 = new aws.EC2()
     ec2.allocateAddress {}, (err, response) ->
       if err
         msg.send err
```